universities/migrations/0002_auto_20191015_1023.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('universities', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='university',
+            options={'ordering': ('-score', 'id'), 'verbose_name': 'University', 'verbose_name_plural': 'Universities'},
+        ),
```

Comment:
if code has been modified but migration not applied, this may change order of universities displayed on site. This ordering is a touchy subject regarding our customer.

Reply:
If this is the case, we have a real consistency issue... but i don't think this will impact production. @sampaccoud WDYT?

Reply:
I would say that the `ordering` meta option is a default clause that is appended to all queries that do not have a specific `order_by` clause... and that it has no effect on the database indexing.

Reply:
I do also think so. Hence this migration will have no impact on the production.

Reply:
Ok fact checked it against [Django's documentation](https://docs.djangoproject.com/en/2.2/ref/models/options/#order-with-respect-to). The option that adds a column in `order_with_respect_to`. the `ordering` option only impacts queries so there is no risk if impacting the current behavior in production.